src/api/remote.rs

```diff
@@ -214,6 +257,55 @@ impl TryFrom<String> for PollInterval {
     }
 }
 
+/// Specificies if we want to scan branches
+#[derive(Debug, Clone, PartialEq, Eq, Display, Deserialize, Serialize, new)]
+pub enum BranchImportStrategy {
+    /// Scanning branches is not allowed
+    Disabled,
+    /// Scanning branches is allowed
+    Enabled,
+}
```

Comment:
Make sure to derive `Copy` for any type that can be made `Copy`: https://rust-lang.github.io/api-guidelines/interoperability.html?highlight=interop#types-eagerly-implement-common-traits-c-common-traits